source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,43 @@
+---
+title: Direct Debit 
+weight: 75
+---
+
+# GOV.UK Pay Variable Direct Debit
+
+Before you read this documentation, you should be familiar with how the
+[GOV.UK Pay API enables users to make
+payments](/payment_flow_overview/#payment-flow). Your contract with GOV.UK Pay
+should also cover Direct Debit, and you should have a merchant account with
+GoCardless. 
```

Comment:
I think we should delete this sentence entirely (Your contract with GOV.UK Pay
should also cover Direct Debit, and you should have a merchant account with
GoCardless.) 
It doesn't explain how someone might've managed to get a contract which covers DD; also the introduction of GC is a little abrupt. 

If someone requests a DD sandbox account, the Pay team can start talking to them about the amended contract then - so we don't need to mention the contract.

Reply:
OK, deleted, no sweat